.husky/install.mjs

```diff
@@ -3,4 +3,4 @@ if (process.env.NODE_ENV === 'production' || process.env.CI === 'true') {
   process.exit(0)
 }
 const husky = (await import('husky')).default
-console.log(husky.install())
+console.log(husky())
```

Comment:
I had to delete this very line of code in my branch to be able to commit changes. Just FYI